core/src/main/scala/org/genivi/sota/core/daemon/TreehubCommitListener.scala

```diff
@@ -61,16 +62,16 @@ class TreehubCommitListener(db: Database, updateService: UpdateService, tufClien
 
   private def publishToTuf(event: TreehubCommit, pid: PackageId): Future[Unit] = {
     val targetMetadata = for {
-      hardwareId ← event.refName.refineTry[ValidHardwareIdentifier]
```

Comment:
I can make them all be ← if you prefer, it just looked weird to be half unicode half ascii art.